src/test/java/com/nulabinc/zxcvbn/MatchingTest.java

```diff
@@ -443,11 +474,12 @@ private void testRegexMatching(String year) throws Exception {
 
         @Test
         public void testOmnimatch() throws Exception {
-            assertEquals(0, new Matching(new ArrayList<String>()).omnimatch("").size());
+            Context context = new ZxcvbnBuilder().buildContext();
+            assertEquals(0, new Matching(context, new ArrayList<String>()).omnimatch("").size());
             String password = "r0sebudmaelstrom11/20/91aaaa";
-            List<Match> matches = new Matching(new ArrayList<String>()).omnimatch(password);
+            List<Match> matches = new Matching(context, new ArrayList<String>()).omnimatch(password);
             Map<Pattern, Integer[]> testMatches = new HashMap<>();
-            testMatches.put(Pattern.Dictionary, new Integer[]{0, 6});
+            //testMatches.put(Pattern.Dictionary, new Integer[]{0, 6});
```

Comment:
Why is this commented out? Did you forget?

Reply:
@vvatanabe It looks like the value put in the commented-out line is overwritten by the next line.
```
            testMatches.put(Pattern.Dictionary, new Integer[]{7, 15});
```
So I commented out that line.